### PR TITLE
Changing headers to sentence case - #1

### PR DESCRIPTION
**Pull request by apinnick**
I will create separate PRs so that this change is easier to review.

* [ ] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [ ] Foreman 3.9/Katello 4.11 (planned Satellite 6.15)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* We do not accept PRs for Foreman older than 3.3.

### CODE REVIEW

File: guides/common/modules/con_administering-hosts.adoc [hunk 1]
```diff
@@ -1,4 +1,5 @@
 [id="Administering_Hosts_{context}"]
-= Administering Hosts
+= Administering hosts
+
```
Comment (maximiliankolb) [inline guides/common/modules/con_administering-hosts.adoc:3]:
```suggestion
```

File: guides/common/modules/con_configuring-external-authentication.adoc [hunk 2]
```diff
@@ -18,7 +18,7 @@ For example, to check if entries for `puppet`, `{apache-user}`, `foreman` and `f
 # cat /etc/group | grep 'puppet\|{apache-user}\|foreman\|foreman-proxy'
 ----
 
-.Scenarios for Configuring External Authentication
+.Scenarios for configuring-external-authentication
```
Comment (maximiliankolb) [inline guides/common/modules/con_configuring-external-authentication.adoc:21]:
```suggestion
.Scenarios for configuring external authentication
```

File: guides/common/modules/con_content-filter-overview.adoc [hunk 3]
```diff
@@ -23,15 +23,15 @@ Use this filter when you want to use most of a particular content repository but
 The filter uses all content in the repository except for the content you select.
 |===
 
-.Include and Exclude Filter Combinations
+.Include and Exclude filter-combinations
```
Comment (maximiliankolb) [inline guides/common/modules/con_content-filter-overview.adoc:26]:
```suggestion
.Include and Exclude filter combinations
```

File: guides/common/modules/con_converting-a-host-to-rhel.adoc [hunk 4]
```diff
@@ -41,7 +41,7 @@ For more information, see https://access.redhat.com/documentation/en-us/red_hat_
 * Ensure that you have enabled and synchronized Red Hat repositories in {Project} for the minor {RHEL} version to which you convert your hosts.
 For more information, see {ContentManagementDocURL}Enabling_Red_Hat_Repositories_content-management[Enabling Red Hat Repositories] and {ContentManagementDocURL}Synchronizing_Repositories_content-management[Synchronizing Repositories] in _{ContentManagementDocTitle}_.
 
-.High-Level Conversion Steps
+.High-level-conversion-steps
```
Comment (maximiliankolb) [inline guides/common/modules/con_converting-a-host-to-rhel.adoc:44]:
```suggestion
.High-level conversion steps
```

File: guides/common/modules/con_host-management-and-monitoring-using-cockpit.adoc [hunk 5]
```diff
@@ -1,5 +1,5 @@
 [id="Host_Management_and_Monitoring_Using_Cockpit_{context}"]
-= Host Management and Monitoring Using {CockpitTitle}
+= Host management and monitoring using {CockpitTitle}
```
Comment (maximiliankolb) [inline guides/common/modules/con_host-management-and-monitoring-using-cockpit.adoc:2]: I am wondering if we should keep "SmartProxyServerTitle" and "CockpitTitle"? Maybe those attributes are no longer is use after this PR.

File: guides/common/modules/con_pull-based-rex-transport-tuning.adoc [hunk 6]
```diff
@@ -1,5 +1,5 @@
 [id="Pull_based_REX_transport_tuning_{context}"]
-= Pull-Based REX Transport Tuning 
+= Pull-Based REX transport tuning
```
Comment (maximiliankolb) [inline guides/common/modules/con_pull-based-rex-transport-tuning.adoc:2]:
```suggestion
= Pull-based REX transport tuning 
```

File: guides/common/modules/con_host-management-and-monitoring-using-cockpit.adoc (hunk 5 above)
Comment (apinnick) [reply guides/common/modules/con_host-management-and-monitoring-using-cockpit.adoc:2]: Good point. The attributes should be reviewed from time to time.

**Review by maximiliankolb — APPROVED** [review]
Thanks, LGTM!

**Comment by maximiliankolb** [comment]
@apinnick You can mark this "ready" and then I'll merge :)

**Comment by maximiliankolb** [comment]
Cherry-picked to 3.9:
   5786c16dc..e1a397bc8  3.9 -> 3.9

**Pull request merged**